modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPUtils.macro

```diff
@@ -17,4 +17,11 @@ definition {
 		}
 	}
 
+	@summary = "Click on kebab from an organization with Awaiting Finance Approval status"
+	macro openMenu {
+		Click(
+			key_name = ${nameMenu},
```

Comment:
Hi @juliosilveira77, could you please rename the variable `nameMenu` to `itemName`? Thanks!